examples/federated_learning/surface_defect_detection/training_worker/interface.py

```diff
@@ -21,6 +21,58 @@
 os.environ['BACKEND_TYPE'] = 'KERAS'
 
 
+import torch
```

Comment:
This looks like the Pytorch framework used, but keras is defined in the Env variable.

Reply:
surface_defect_detection is using Keras, and we will perform it in a similar way.